components/colorwheel/index.css

```
@@ -11,32 +11,33 @@
  * governing permissions and limitations under the License.
  */
 
-/* Windows High Contrast Mode */
-@media (forced-colors: active) {
-	.spectrum-ColorWheel {
-		--highcontrast-colorwheel-border-color-disabled: GrayText;
-		--highcontrast-colorwheel-fill-color-disabled: Canvas;
+.spectrum-ColorWheel {
+	--spectrum-colorwheel-width: var(--spectrum-color-wheel-width);
+	--spectrum-colorwheel-min-width: var(--spectrum-color-wheel-minimum-width);
+	--spectrum-colorwheel-height: var(--spectrum-color-wheel-width);
 
-		forced-color-adjust: none;
-	}
-}
+	/* @TODO: leveraging the rgb token in this case to achieve the desired border color implementation as rgb + opacity are required by the `rgba` function. */
+	--spectrum-colorwheel-border-color-rgb: var(--spectrum-gray-1000-rgb);
+
+	/* @TODO: using explicit value as `color-wheel-border-opacity` is undefined. */
+	--spectrum-colorwheel-border-opacity: 0.1;
```

Was this defined in a recent tokens update? https://github.com/adobe/spectrum-tokens/blob/5f67e9c2d0fe52d3f488cead97f41b427f327215/packages/tokens/src/color-component.json#L481

beta.55 maybe?

```suggestion
--spectrum-colorwheel-border-opacity: var(--spectrum-color-wheel-border-opacity);
```